torch/nn/modules/activation.py

```diff
@@ -891,13 +891,12 @@ def forward(self, query, key, value, key_padding_mask=None,
           If a ByteTensor is provided, the non-zero positions will be ignored while the position
           with the zero positions will be unchanged. If a BoolTensor is provided, the positions with the
           value of ``True`` will be ignored while the position with the value of ``False`` will be unchanged.
-        - attn_mask: 2D mask :math:`(L, S)` where L is the target sequence length, S is the source sequence length.
-          3D mask :math:`(N*num_heads, L, S)` where N is the batch size, L is the target sequence length,
-          S is the source sequence length. attn_mask ensure that position i is allowed to attend the unmasked
-          positions. If a ByteTensor is provided, the non-zero positions are not allowed to attend
-          while the zero positions will be unchanged. If a BoolTensor is provided, positions with ``True``
-          is not allowed to attend while ``False`` values will be unchanged. If a FloatTensor
-          is provided, it will be added to the attention weight.
+        - attn_mask: Either a 2D mask :math:`(L, S)` or a 3D mask :math:`(N*num\_heads, L, S)` where N is the
```

Comment:
`num\_heads`?

Reply:
We need to escape the underscore otherwise it looks like this: https://pytorch.org/docs/master/generated/torch.nn.MultiheadAttention.html

![image](https://user-images.githubusercontent.com/8558782/87343518-d2c53480-c501-11ea-8c65-19d7bbd25324.png)